OctoAwesome/OctoAwesome/OrientationFlags.cs

```diff
@@ -60,7 +60,7 @@ public enum OrientationFlags : byte
         #region Sides
 
         /// <summary>
-        /// Beschreibt die komplette Seite von -X
+        /// Describes the complete side on von -X
```

Comment:
```suggestion
        /// Describes the complete side from -X
```

Who doesn't know the common english word "von" :)